client/perffarm-client.py

```diff
@@ -70,8 +73,8 @@
         PGBENCH_CONFIG['results_dir'] = OUTPUT_DIR
         runner.register_config('pgbench-basic',
                                'pgbench',
-                               repository.current_branch(),
-                               repository.current_commit(),
```

Comment:
no need to change this, I uploaded a zip file that contains the .git, you could extract that in the designated folder.